### PR TITLE
FAST #5856 for wc - rows/cols for fluent-text-area

### PR DESCRIPTION
**Pull request by vnbaaij**
## Previous Behavior

<fluent-text-area> does not respond to setting row and/or cols attribute

## New Behavior

<fluent-text-area> does respond to setting row and/or cols attribute
## Related Issue(s)

- Fixes #

### CODE REVIEW

**Review by chrisdholt — APPROVED** [review]

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/b922d4bbfce680ac0a8ec46b73e30958c7fa3b09'>b922d4bbfce680ac0a8ec46b73e30958c7fa3b09</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by Ogglas** [comment]
Relates to: https://github.com/microsoft/fluentui/issues/27842

File: change/@fluentui-web-components-c8b4d98c-6c11-4728-b8db-a5e5e77f78da.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fix(web-components): ensures that text area size responds to rows and cols attributes",
+  "packageName": "@fluentui/web-components",
+  "email": "chhol@microsoft.com",
```
Comment (chrisdholt) [inline change/@fluentui-web-components-c8b4d98c-6c11-4728-b8db-a5e5e77f78da.json:5]:
```suggestion
  "email": "vnbaaij@outlook.com",
```

**Comment by Ogglas** [comment]
@chrisdholt @micahgodbolt Can this PR be merged?

**Comment by chrisdholt** [comment]
@vnbaaij can you try and update/sync your branch with the latest from Fluent? Seems like there is likely a rebase issue happening causing issues with the build.

File: packages/web-components/src/text-area/text-area.styles.ts [hunk 2]
```diff
@@ -54,6 +54,14 @@ export const textAreaStyles: (context: ElementDefinitionContext, definition: Fou
     :host(.resize-vertical) .control {
       resize: vertical;
     }
+
+    :host([cols]) {
```
[the remark this answers is not in the document]
Comment (vnbaaij) [reply packages/web-components/src/text-area/text-area.styles.ts:58]: I just copied what you did in https://github.com/microsoft/fast/pull/5856

**Comment by Ogglas** [comment]
Is anything more needed to merge this?

**Comment by chrisdholt** [comment]
@micahgodbolt any idea why the build isn't running here at all?

**Comment by chrisdholt** [comment]
@JustSlone The react build hasn't been running on @vnbaaij's PR since it was opened - is there perhaps some issue that can occur for contributors?

**Pull request merged**